consensus/src/liveness/round_state.rs

```diff
@@ -254,8 +258,8 @@ impl RoundState {
     /// Notify the RoundState about the potentially new QC, TC, and highest committed round.
     /// Note that some of these values might not be available by the caller.
     pub fn process_certificates(&mut self, sync_info: SyncInfo) -> Option<NewRoundEvent> {
-        if sync_info.highest_ordered_round() > self.highest_committed_round {
-            self.highest_committed_round = sync_info.highest_ordered_round();
+        if sync_info.highest_commit_round() > self.highest_committed_round {
```

Comment:
this is wrong, we should take the sync_info.highest_ordered_round and rename self.highest_committed_round to self.highest_ordered_round